.github/workflows/ci.yaml

```
@@ -102,7 +102,7 @@ jobs:
           juju model-defaults logging-config='<root>=INFO; unit=DEBUG'
           juju add-model test
       - name: Terraform deploy
-        working-directory: ./terraform/tests/
+        working-directory: ./terraform/modules/
```

There's an inconsistency in this working directory path and the former one :)
```suggestion
        working-directory: ./terraform/modules
```